lib/utils.js

```diff
@@ -19,6 +19,7 @@ var etag = require('etag');
 var proxyaddr = require('proxy-addr');
 var qs = require('qs');
 var querystring = require('querystring');
+var flatten = require('array-flatten');
```

Comment:
Typically you would just add it to your `package.json` and just push a new commit to the branch that this PR came from :)